doc/content/the-things-stack/concepts/server-addresses/_index.md

```diff
@@ -54,7 +54,7 @@ The CUPS gateway address is a combination of the **protocol** (https), the **ser
 
 **Example 1**: The <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Community Edition CUPS address is <code data-content="cluster-address">https://<span data-content="cluster-id"></span>.cloud.thethings.network:443</code>.
 
-**Example 2**: The CUPS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>, where `tenant` should be replaced with your Tenant ID.
+**Example 2**: The CUPS address for an <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>.
```

Comment:
```suggestion
**Example 2**: The CUPS address for a <code data-content="cluster-address"><span data-content="cluster-id"></span></code> Cloud tenant is <code data-content="cluster-address">https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443</code>.
```

Reply:
@nejraselimovic Thanks for catching up on the typo.